botx/exceptions.py

```diff
@@ -63,3 +63,7 @@ class ServerUnknownError(BotXException):
 
     #: host that is unregistered.
     host: str
+
+
+class BotXAPIRouteDeprecated(BotXAPIError):
+    """Raised if API route was deprecated."""
```

Comment:
Add message_template text. unable to send {method} {url} to BotX API ({status}) is too common.

Reply:
done